python/vmaf/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 __copyright__ = "Copyright 2016-2020, Netflix, Inc."
 __license__ = "BSD+Patent"
-
+__version__ = "1.5.1"
```

Comment:
In this case, there are two places where the version is specified: here, and in the VERSION file in the root directory. We can probably remove the outer file. Or you prefer to keep both?

Reply:
It's necessary for the version specification to at least be somewhere inside the Python repository once you try to pip-install it. So I would suggest keeping it only there, or bumping it in two places if you have an easy way to do that via a release script.